apps/web/lib/team/[slug]/[type]/getServerSideProps.tsx

```diff
@@ -75,6 +75,22 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   });
 
   if (!eventData) {
+    const eventTypeByPreviousSlug = await prisma.eventType.findUnique({
+      where: {
+        teamId_previousSlug: {
+          teamId: team.id,
+          previousSlug: meetingSlug,
+        },
+      },
+    });
+    if (eventTypeByPreviousSlug) {
+      return {
+        redirect: {
+          destination: `/team/${teamSlug}/${eventTypeByPreviousSlug.slug}`,
+          permanent: true,
+        },
+      };
+    }
```

Comment:
If the `event-type` is not found using the slug, try to use `previousSlug` to find the `event-type` if found we should redirect to the slug of the `eventTypeByPreviousSlug`